rotate.go

```diff
@@ -388,6 +455,7 @@ func (r *Rotator) SetSecret(ctx context.Context, event map[string]string) error
 		Step: "setSecret",
 		Time: r.startTime,
 	})
+	debugSecret("db credentials: %+v", creds)
```

Comment:
Put it right after `creds := db.NewPassword{`, else this debug might not occur if the func returns early on the new edge case handling.

Reply:
updated